x-pack/plugins/alerting/server/routes/get_rule_alert_summary.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IRouter } from 'kibana/server';
+import { schema } from '@kbn/config-schema';
+import { ILicenseState } from '../lib';
+import { GetAlertInstanceSummaryParams } from '../alerts_client';
+import { RewriteRequestCase, RewriteResponseCase, verifyAccessAndContext } from './lib';
+import {
+  AlertingRequestHandlerContext,
+  BASE_ALERTING_API_PATH,
+  AlertInstanceSummary,
+} from '../types';
+
+const paramSchema = schema.object({
+  id: schema.string(),
+});
+
+const querySchema = schema.object({
+  date_start: schema.maybe(schema.string()),
+});
+
+const rewriteReq: RewriteRequestCase<GetAlertInstanceSummaryParams> = ({
+  date_start: dateStart,
+  ...rest
+}) => ({
+  ...rest,
+  dateStart,
+});
+const rewriteBodyRes: RewriteResponseCase<AlertInstanceSummary> = ({
+  alertTypeId,
+  muteAll,
+  statusStartDate,
+  statusEndDate,
+  errorMessages,
+  ...rest
+}) => ({
+  ...rest,
+  rule_type_id: alertTypeId,
+  mute_all: muteAll,
+  status_start_date: statusStartDate,
+  status_end_date: statusEndDate,
+  error_messages: errorMessages,
```

Comment:
There's a `lastRun` field inside `AlertInstanceSummary`. Does that need to be rewritten as `last_run`?

Reply:
Interesting, I'm not sure how TS didn't catch this, thanks! Fixed in https://github.com/elastic/kibana/pull/93980/commits/ca2512a0d02218af7f02180d64aef3cd76970354